Exiled.CustomItems/API/Features/CustomItem.cs

```diff
@@ -468,9 +468,38 @@ public virtual uint Spawn(IEnumerable<SpawnPoint> spawnPoints, uint limit)
 
                 spawned++;
 
-                Spawn(spawnPoint.Position.ToVector3, out _);
+                if (spawnPoint is DynamicSpawnPoint dynamicSpawnPoint && dynamicSpawnPoint.Location == SpawnLocation.InsideLocker)
+                {
+                    for (int i = 0; i < 50; i++)
+                    {
+                        Locker locker =
+                            LockerManager.singleton.lockers[
+                                Instance.Rng.Next(LockerManager.singleton.lockers.Length)];
+                        if (locker._itemsToSpawn == null)
+                        {
+                            Log.Debug($"{nameof(Spawn)}: Invalid locker location. Attempting to find a new one..", Instance.Config.Debug);
+                            continue;
+                        }
+
+                        LockerChamber chamber = locker.chambers[Instance.Rng.Next(Mathf.Max(0, locker.chambers.Length - 1))];
+
+                        foreach (Locker.ItemToSpawn item in locker._itemsToSpawn.ToList())
+                        {
+                            if (item._pos == chamber.spawnpoint.position)
```

Comment:
Why do you prevent normal items from spawning if they're in the same location of the custom ones?

Reply:
Because then you would end up with more than 1 item in a small locker, which can cause physics issues that cause one (or sometimes both) item(s) to bounce out of the locker container.

By small locker, I am referring to the big cabinets of small, locked locker chambers that normally contain items like keycards, ammo and grenades.